.github/workflows/dockerfile-llb-frontend.yaml

```diff
@@ -0,0 +1,36 @@
+name: tools/dockerfile-llb-frontend
+
+on:
+  push:
+    branches: [stable]
+    paths:
+      - 'tools/dockerfile-llb-frontend/**'
+      - '.github/workflows/dockerfile-llb-frontend.yaml'
+
+jobs:
+  plugin-push:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to Registry
+        uses: docker/login-action@v2
+        with:
+          registry: index.unikraft.io
+          username: ${{ secrets.REG_USERNAME }}
+          password: ${{ secrets.REG_TOKEN }}
+
+      - name: Build OCI image for the LLB plugin
+        uses: docker/build-push-action@v4
+        with:
+          context: ./tools/dockerfile-llb-frontend
+          file: ./tools/dockerfile-llb-frontend/Dockerfile
+          tags: index.unikraft.io/kraftkit.sh/llb:latest
+          push: true
```

Comment:
Same with pushing w.r.t. to push events:
```diff
- push: true
+ push: ${{ github.event_name == 'push' }}
```

Reply:
Done